config/tokenConfig.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const Joi = require('joi');
+const models = require('../models');
+
+const TOKEN_CONFIG = {
+    username: Joi.string(),
+    buildId: Joi.reach(models.build.base, 'id').required(),
+    jobId: Joi.reach(models.job.base, 'id').required(),
+    eventId: Joi.reach(models.event.base, 'id'),
+    isPR: Joi.boolean().required().default(false),
```

Comment:
we can have default true, it's safer on auth levels. in-case there is a mistake  ?

Reply:
ok, should this always be true, meaning all requests are PR's?

Reply:
no caller should take care of passing it ?

Reply:
does it have to have a `default` value ?